examples/tutorials/api/astro_dark_matter.py

```diff
@@ -244,7 +245,8 @@
 massDM = 10 * u.TeV
 diff_flux = DarkMatterDecaySpectralModel(mass=massDM, channel=channel)
 int_flux = (
-    jfact_decay * diff_flux.integral(energy_min=0.1 * u.TeV, energy_max=10 * u.TeV)
+    jfact_decay.to_value("GeV cm-2")
```

Comment:
Wouldn't it be better to directly rewrite the definition of `jfact_decay` to `dfact` in the tutorial notebook? Anyway, this is just a nomenclature suggestion.